client/galaxy/scripts/onload.js

```diff
@@ -170,11 +170,27 @@ $(document).ready( function() {
     if (et){
         et = TOURS.hooked_tour_from_data(et);
         if (et && et.steps){
-            var tour = new Tour(_.extend({
-                steps: et.steps,
-            }, TOURS.tour_opts));
-            tour.init();
-            tour.restart();
+            var in_iframe;
+            try {
+                if (window.self !== window.top){
+                    in_iframe = true;
+                } else {
+                    in_iframe = false;
+                }
+            } catch (e) {
+                in_iframe = false;
+            }
+            if (!in_iframe){
```

Comment:
Not super thrilled about the level of nitpicking here.  I do find it pretty amusing that guerler said "I think it is" [the same], and you said "I think you can do...", where the code I wrote is fairly explicit and self-documenting.  That was kinda the point.  I'll go open another PR to change this not-important-at-all function, though.